correios/client.py

```diff
@@ -99,6 +107,24 @@ def __init__(
 
         self.model_builder = ModelBuilder()
 
+    def _exception_call(self, exception):
+        message = str(exception)
+        exception = ClientError(message)
+
+        if "autenticacao" in message:
+            message = "Authentication error for user {}".format(self.username)
+            exception = AuthenticationError(message)
```

Comment:
You should use `elif` or just return the exception in the checks below. the way the code is now looks like more than one condition can be true at the same time which I think is not the case.

IMHO the final code should look something like this:

```python
if expr:
    ...
    return XError(...)

if expr2:
    ...
    return YException(...)

...

return ClientError(str(exception))
```